### PR TITLE
[FLINK-3583] User configuration visible in webdashboard when job is running

### PR DESCRIPTION
**Pull request by fijolekProjects**
Before:
no-configuration-when-job-running

After;
after-changes

### CODE REVIEW

File: flink-runtime/src/main/java/org/apache/flink/runtime/executiongraph/ExecutionGraph.java [hunk 1]
```diff
@@ -306,6 +306,12 @@ public ExecutionGraph(
 		this.timeout = timeout;
 
 		this.restartStrategy = restartStrategy;
+
```
Comment (aljoscha) [inline flink-runtime/src/main/java/org/apache/flink/runtime/executiongraph/ExecutionGraph.java]: @StephanEwen @uce was there a reason for not reading the executionConfig at this point already?

**Comment by aljoscha** [comment]
Thanks for your PR! I'll look into whether loading the ExecutionConfig earlier breaks anything and then merge it.

**Comment by StephanEwen** [comment]
I think that #1583 also addresses parts this, in a more generic fashion.
Reading the config earlier should not be a problem.

I would vote to get first #1583 and then from this pull request only the web frontend changes.

**Comment by aljoscha** [comment]
Ok, makes sense.

**Comment by fijolekProjects** [comment]
Thanks, I removed backend changes

**Comment by aljoscha** [comment]
Hi @fijolekProjects,
could you please confirm that everything works now that #1583 is merged. If yes, then I would like to merge this one as well.

**Comment by fijolekProjects** [comment]
Hi @aljoscha I can confirm that after merging #1583 this change looks good

**Comment by aljoscha** [comment]
Thanks a lot! I merged it.

Could you please close this PR if github does not do so automatically.

**Comment by fijolekProjects** [comment]
Thanks @aljoscha

**Pull request closed without merging**